<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zoesuan</class>
 <widget class="QWidget" name="zoesuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房贷计算器</string>
  </property>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>711</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>690</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>贷款类别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>商业贷款</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>公积金贷款</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>组合型贷款</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>480</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>693</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>计算方式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>根据贷款总额计算</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>根据面积，单价计算</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>480</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>372</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>贷款总额：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>万</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>75</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>693</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>按揭年数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>25年（300期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24年（288期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23年（276期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22年（264期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21年（252期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20年（240期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19年（228期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18年（216期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17年（204期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16年（192期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15年（180期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14年（168期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13年（156期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12年（144期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11年（132期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10年（120期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9年（108期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8年（96期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7年（84期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6年（72期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5年（60期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4年（48期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3年（36期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2年（24期）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1年（12期）</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>480</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>240</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>贷款利率:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>65</x>
          <y>208</y>
          <width>461</width>
          <height>51</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>25年（300期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24年（288期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23年（276期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22年（264期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21年（252期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20年（240期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19年（228期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18年（216期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17年（204期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16年（192期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15年（180期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14年（168期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13年（156期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12年（144期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11年（132期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10年（120期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9年（108期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8年（96期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7年（84期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6年（72期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5年（60期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4年（48期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3年（36期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2年（24期）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1年（12期）</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>50</width>
          <height>419</height>
         </rect>
        </property>
        <property name="text">
         <string>按揭年数：</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>364</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>还款方式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>等额本息</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>等额本金</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>693</width>
      <height>68</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>开始计算</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>清空重填</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_15" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_16" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_17" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_18" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_19" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_20" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
